.meta/sources/prometheus.toml

```diff
@@ -0,0 +1,62 @@
+[sources.prometheus]
+beta = true
+common = false
+delivery_guarantee = "best_effort"
+function_category = "receive"
+guides = []
+output_types = ["metric"]
+resources = []
+through_description = "the Prometheus text exposition format"
+
+[sources.prometheus.options.host]
+type = "string"
+common = true
+null = false
+examples = ["http://localhost:9090"]
+description = "Host address to scrape metrics from."
+
+[sources.prometheus.options.scrape_interval_secs]
+type = "int"
+common = true
+null = false
+examples = [1]
+description = "The interval between scrapes in seconds."
+
+[[sources.prometheus.output.metric.examples]]
+label = "Counter"
+input = """\
+# TYPE promhttp_metric_handler_requests_total counter\
+promhttp_metric_handler_requests_total{code="200"} 100\
+"""
+output = """\
+{
+  "name": "promhttp_metric_handler_requests_total",
+  "kind": "absolute",
+  "timestamp": "2019-05-02T12:22:46.658503Z" // current time / time ingested
+  "tags": {
+    "code": "200"
+  },
+  "value": {
```

Comment:
Is this correct? I thought the format was:

```json
{
  "name": "promhttp_metric_handler_requests_total",
  "kind": "absolute",
  "timestamp": "2019-05-02T12:22:46.658503Z" // current time / time ingested
  "tags": {
    "code": "200"
  },
  "counter": {
    "value": 100.0
  }
}
```

Reply:
Nice catch! I've updated metric [model docs](https://deploy-preview-1264--vector-project.netlify.com/docs/about/data-model/metric/#type) to reflect our actual [serialization format](https://github.com/timberio/vector/blob/master/src/sinks/console.rs#L117), which looks like `"value":{"type":"counter","value":100.0}`.

Reply:
Thanks. Doesn't this deviate from what's defined in the [`event.proto` file](https://github.com/timberio/vector/blob/master/proto/event.proto)? It appears that the type is the root key and the value is nested.

Reply:
Thanks for pointing. I've renamed `metric` field into `value` in proto. It should be _conceptually_ the same now. There is a difference in encoding though because we are using [internally tagged](https://serde.rs/enum-representations.html#internally-tagged) representations in serialization.